static/lang/re-en.json

```
@@ -5623,6 +5623,7 @@
                 "Color": "Select a Color.",
                 "ConsciousMind": "Select a conscious mind.",
                 "CreatureSize": "Select a creature size.",
+                "DamageDice": "Select a number of damage dice.",
```

```suggestion
                "DamageDice": "Select a number of damage dice.",
```
Maybe WeaponDamageDice and "number of weapon damage dice" would be better? Since it's more clear what is meant.

---

Yeah, that makes sense. Will get that done when I rebase.